flow/src/main/scala/org/alephium/flow/network/DiscoveryServerState.scala

```diff
@@ -143,14 +143,12 @@ trait DiscoveryServerState extends SessionManager {
       case Some(until) => unreachables.put(remote, until + Duration.ofMinutesUnsafe(1))
       case None        => unreachables.put(remote, TimeStamp.now().plusMinutesUnsafe(1))
     }
-    log.debug(s"Set $remote as unreachable")
```

Comment:
Git nazi here: you shouldn't have add this in your previous commit if you remove it right away.